draft-ietf-jsonpath-base.md

```diff
@@ -256,7 +256,7 @@ Nodelist:
   does not require or assume any particular representation.
 
 Parameter:
-: Formal parameter that can take Arguments (actual parameters) in a function-expression.
+: Formal parameter (of a function) that can take function arguments (actual parameters) in a function-expression.
```

Comment:
Let's not suggest the a single parameter can take multiple arguments:
```suggestion
: Formal parameter (of a function) that can take a function argument (actual parameter) in a function-expression.
```

Reply:
Can you copy over this change to #428?
(I can't, because of the weird issue with PRs from "main" on "main".)